### PR TITLE
Lighter GUI

### PR DESCRIPTION
**Pull request by IanDuncanT**

### CODE REVIEW

**Comment by IanDuncanT** [comment]
Merge branch 'master' into master is a pull request conflict fix. You should review to make sure I am not overriding your code.

File: yapi.py [hunk 1]
```diff
@@ -28,16 +38,55 @@ def print_commands_allowed():
             print("\t - {} \n\t\t python yapi.py {}".format(
                 options[option][1], option))
 
-
 def argumentError(arg):
     """Argument error."""
     print("The argument {} isn't allowed,".format(arg.upper()))
     print_commands_allowed()
 
+def build(app):
```
Comment (Wabri) [inline yapi.py]: I have transfered this functionality on this file from your precedent PR -> [user_interface.py](https://github.com/Wabri/YAPI/blob/master/user_interface.py)

File: .travis.yml [hunk 2]
```diff
@@ -9,10 +9,8 @@ matrix:
...
       dist: trusty
-    - python: 3.3
+    - python: 3.5
       dist: trusty
-    - python: 3.2
-      dist: trusty
-    - python: 2.7
+    - python: 3.4
       dist: trusty
 script: python yapi.py install test
```
[the remark this answers is not in the document]
Comment (IanDuncanT) [reply .travis.yml]: I was just recieving errors on the requirements.txt and needed to test. If there could be a way to not install for travis and install for users when pulling yapi, then we should change it to that.

**Comment by IanDuncanT** [comment]
I believe I have fixed what you have asked for, with the exception of line length which I am working on now.

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (IanDuncanT) [reply .travis.yml]: I will add more versions, but remove toga from the requirements for now, however we should find a way to have a different version for users and travis.

File: yapi.py [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
 
-import cache_manager
-import script_runner
-import sys  # Make System Calls
+import glob                   #UNIX Pathname Expansion
+import os                     #Operating System Calls
+import pickle                 #Serialization and Deserialization of .bin files
+import subprocess             #Run Bash Commands
+import sys                    #Make System Calls
+import time                   #Time Library
```
[the remark this answers is not in the document]
Comment (IanDuncanT) [reply yapi.py]: I removed them

File: yapi.py [hunk 4]
```diff
@@ -28,16 +38,55 @@ def print_commands_allowed():
...
+    for package_counter in packages:
+        packageString += "{:>2}) {} - {}".format(
+            package_counter,
+            packages[package_counter][0].capitalize(),
+            packages[package_counter][1])
+    box = toga.Box()
+    listBox = toga.Box()
+    listLabel = toga.Label('Packages Available: ' + packageString)
+    packageBox = toga.Box()
+    packageLabel = toga.Label('Package To Install:', style=Pack(text_align=RIGHT))
```
[the remark this answers is not in the document]
Comment (IanDuncanT) [reply yapi.py]: I removed that part and it looks fine without it.

**Comment by IanDuncanT** [comment]
I am going to merge this as one commit.

**Pull request merged**